test/models/pawn_test.rb

```diff
@@ -118,4 +118,31 @@ class PawnTest < ActiveSupport::TestCase
     assert_not move
   end
 
+  test "update en passant status (true case)" do
+    game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+    pawn = game.pieces.where(:pos_x => 4).where(:pos_y => 1).first    
+    move = pawn.update_en_passant_status(4, 3)
+    assert pawn.en_passant  
+  end
+
+  test "update en passant status (false case)" do
+    game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+    pawn = game.pieces.where(:pos_x => 4).where(:pos_y => 1).first    
+    move = pawn.update_en_passant_status(4, 2)
+    assert_not pawn.en_passant  
+  end
+
+  test "can capture en passant (true case)" do
```

Comment:
I've been having trouble figuring out why this isn't working. I think it hass something to do with checking if the game and the piece were updated at the same time. Are timestamps more accurate than they show? As in, they display to the second, but in reality are they actually recorded more accurately? I was printing their timestamps to check, but they would probably show up as the same even if they weren't linked, because the test happens to fast anyways